TackleTabby/Assets/Scripts/MainCharacter/AnimationEventHolder.cs

```diff
@@ -0,0 +1,21 @@
+using UnityEngine;
+
+public class AnimationEventHolder : MonoBehaviour
+{
+    [SerializeField]
+    private CaughtFishPopup fishPopUp;
+
+    [SerializeField]
+    private AudioHandler audioHandler;
```

Comment:
these don't follow our naming convention
```suggestion
    [SerializeField]
    private CaughtFishPopup FishPopUp;

    [SerializeField]
    private AudioHandler AudioHandler;
```